### ISSUE TITLE
incorrect example

### ISSUE THREAD

**Issue by samcoppock** [Sheikh-Alvee/laravel-worldpay#1]
sample code reads 
$key      = config('worldpay.sandbox.client');

it should read
$key      = config('worldpay.sandbox.service');

**Comment by imran-saleem834** [comment]
yeah you've fine, i've not worked on new Laravel Version, that's why i didn't get this done. but will do soon
